src/main/java/com/qaprosoft/carina/demo/operations/InitialPage.java

```diff
@@ -0,0 +1,57 @@
+package com.qaprosoft.carina.demo.operations;
+
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.qaprosoft.carina.core.foundation.utils.Configuration;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+
+public class InitialPage extends AbstractPage{
+
+
+    @FindBy(xpath = "//input[@id='login-username']")
+    public ExtendedWebElement username;
+
+    @FindBy(xpath = "//input[@type='password']")
+    private ExtendedWebElement password;
+
+    @FindBy(xpath = "//input[@value='Next']")
+    private ExtendedWebElement nextButton;
+
+    @FindBy(xpath = "//button[@name='verifyPassword']")
+    private ExtendedWebElement signIn;
+
+    @FindBy(xpath = "//button[@id='uh-search-button']")
+    private ExtendedWebElement searchButton;
+
+    @FindBy(xpath = "//input[@id='uh-search-box']")
+    private ExtendedWebElement searchField;
+
+    public InitialPage(WebDriver driver) {
+        super(driver);
+    }
+
+
+    public void login() {
+        username.type("maxjaycob@yahoo.com");
```

Comment:
please choose another resource and don't put credentials directly into the testclass.
If you wanna to keep existing credentials please:
1) move their declaration into _config.properties
2) encrypt password using https://github.com/qaprosoft/carina-archetype/wiki/Carina-tutorial#security tutorial and existing src/main/resources/crypto.key